### DISCUSSION TITLE
Q:스태틱파일올리기

### DISCUSSION THREAD

**Discussion by oljindudol** [oljindudol/exoman#16] [General]
스태틱파일업로드후 URL따서
외부에서 쓰는방법 있을까요?

**Comment by oljindudol** [comment]
잠깐 해결한거같음

**Reply by yhj9619** [reply]
ㅇㅅㅇ?ㅋ

**Reply by oljindudol** [reply]
스태틱태그 소스에서 한번도안쓰면
스태틱 폴더에 접근못해요?

**Reply by yhj9619** [reply]
정확히 머선소리인지 아리까리한데...
https://nachwon.github.io/django-deploy-4-static/
요렁 거 하는 거 얘긴가여 호에에

**Reply by oljindudol** [reply]
앗 해결하였읍니다
감사합니다
